### PR TITLE
Monero

### PR DESCRIPTION
**Pull request by iliemihai**
Biomedical Romanian dataset :)

### CODE REVIEW

**Comment by skyprince999** [comment]
Hi @iliemihai  - you need to add the Readme file! Otherwise seems good.

**Comment by iliemihai** [comment]
@skyprince999 I will add the README.d for it. Thank you :D

File: datasets/monero/monero.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+
+class MONEROConfig(datasets.BuilderConfig):
+    """BuilderConfig for RONEC dataset"""
+
+    def __init__(self, **kwargs):
+        super(MONEROConfig, self).__init__(**kwargs)
+
+
+class MONERODataset(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/monero/monero.py:62]:
```suggestion
class Monero(datasets.GeneratorBasedBuilder):
```
this must be the camel caser version of the dataset script name `monero`

File: datasets/monero/monero.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    """RONEC dataset"""
```
Comment (lhoestq) [inline datasets/monero/monero.py:63]:
```suggestion
    """MONERO dataset"""
```

File: datasets/monero/monero.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                        last = len(sent)
+                        label = splits[2][2:]
+                        has_started = True
+                    elif splits[2].startswith("I-"):
+                        last = len(sent)
+                    elif splits[2].startswith("O") and begin:
+                        ner.append(label)
+                        start.append(begin)
+                        end.append(last)
+                        has_started = False
```
Comment (lhoestq) [inline datasets/monero/monero.py:189]: You shouldn't remove the IOB tags.

You can get some inspiration from conll2003: https://github.com/huggingface/datasets/blob/master/datasets/conll2003/conll2003.py

**Review by lhoestq — COMMENTED** [review]
Looking pretty good so far. I left a few comments, overall the same ones as for RONEC.

Also it looks like in this PR there are also the files from RONEC and rosent, could you remove them to only keep Monero please ?

**Pull request closed without merging**